networkit/cpp/auxiliary/GlobalState.cpp

```diff
@@ -0,0 +1,146 @@
+// networkit-format

```

Comment:
Maybe this file should not be in `Aux` but at the top level in `networkit/cpp/` since it (potentially) affects all modules.